great_expectations/analytics/actions.py

```diff
@@ -6,6 +6,7 @@
 DATA_CONTEXT_INITIALIZED: Action = Action("data_context.initialized")
 CHECKPOINT_CREATED: Action = Action("checkpoint.created")
 CHECKPOINT_DELETED: Action = Action("checkpoint.deleted")
+CHECKPOINT_RAN: Action = Action("checkpoint.ran")
```

Comment:
ran vs run?

Reply:
every analytics action is named in the past tense, so `checkpoint.ran` is consistent.